contrib/cf-remote/cf_remote/commands.py

```diff
@@ -81,10 +116,17 @@ def install(
     assert not (hubs and clients and package)
     # These assertions are checked in main.py
 
+    # If there are URLs in any of the package strings, downlaod and replace with path:
+    packages = (package, hub_package, client_package)
+    package, hub_package, client_package = _download_urls(packages)
+
+
```

Comment:
One blank line ought to be enough inside functions.

Reply:
Fixed.